$OpenBSD: patch-src_tracker-indexer_tracker-indexer_c,v 1.1 2009/10/09 17:40:12 ajacoutot Exp $

We do not support HAL.
XXX should probably use an ifdef

--- src/tracker-indexer/tracker-indexer.c.orig	Sat Oct  3 18:24:53 2009
+++ src/tracker-indexer/tracker-indexer.c	Sat Oct  3 18:25:02 2009
@@ -1760,7 +1760,7 @@ item_add_or_update (TrackerIndexer        *indexer,
 		/* Service wasn't previously indexed */
 		id = tracker_data_update_get_new_service_id (indexer->private->common);
 		data = tracker_module_metadata_get_hash_table (metadata);
-		udi = tracker_hal_udi_get_for_path (indexer->private->hal, service_path);
+		udi = NULL;
 
 		context = tracker_data_update_metadata_context_new (TRACKER_CONTEXT_TYPE_INSERT,
 								    service, id);
